app/templates/templates/template.hbs

```diff
@@ -4,6 +4,20 @@
 </div>
 <div class="ml-2">
   Capital: {{capitalize model.capitalName}}
+  <select onchange={{action "selectCapital" value="target.value"}}>
+    {{#each capitalBuildings as |capital|}}
+      <option
+        value={{capital.lvl}}
+        selected={{eq capital selectedCapital}}
+        disabled={{gt capital.TPcost (add selectedCapital.TPcost remainingTemplatePoints)}}
+      >
+        {{capitalize capital.name}}
+        {{#if capital.TPcost}}
+          <span class="ml-auto">({{capital.TPcost}} TP)</span>
```

Comment:
We don't need a span ml-auto, since we don't use the bootstrap select (yet).

I need to make a revamp of the graphic design at some point... But not today!